src/hyperlink/strategies.py

```diff
@@ -0,0 +1,292 @@
+# -*- coding: utf-8 -*-
+"""
+Hypothesis strategies.
+"""
+
+try:
+    import hypothesis
+    del(hypothesis)
```

Comment:
Is this some py4 forward compat?? jkjk, but `del` is still a statement in py3 :)

Reply:
Oh, I can't help but use parens, oops.

This was just to separate the "do we have hypothesis?" test from where we import what we need; I was finding it annoying to have one import line up by it's lonesome.  :-?

Reply:
Perhaps you were asking why "do we have hypothesis?" is a question.
It's (perhaps weirdly) a Python 3.4 thing.
Hypothesis supports 2.7 and >=3.5, but not 3.0-3.4.

Reply:
Heh, nope just eyeing the parens.